frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestReviewStatus/ChangeRequestReviewStatus.tsx

```diff
@@ -109,6 +112,12 @@ const ResolveComponent = ({ changeRequest }: IResolveComponentProps) => {
         return <Rejected />;
     }
 
+    if (state === 'Scheduled') {
+        return (
+            <Scheduled scheduledDate={changeRequest.schedule?.scheduledAt} />
```

Comment:
Is the `?` necessary here? Isn't `schedule` required if the state is `Scheduled`? Or are there cases where it might not be present?

```suggestion
            <Scheduled scheduledDate={changeRequest.schedule.scheduledAt} />
```

Reply:
Because `schedule` can be undefined - the ts compiler is complaining. 
I can either:
- make the `scheduledDate` prop with a `?`
- make `changeRequest.schedule!.scheduledAt`
or leave it as is